CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ Compatibility:
 * Fix arity for `Proc` (#2098, @ssnickolay)
 * Check bounds for `FFI::Pointer` accesses when the size of the memory behind is known. 
 * Implement negative line numbers for eval (#1482).
+* Support refinements for `String#to_s` (#2110, @ssnickolay)
```

Comment:
Actually it's for String interpolation, I'll update to that.

Reply:
Thanks!